lizrd/scripts/sync_with_remote.py

```diff
@@ -0,0 +1,63 @@
+import os
+from fabric import Connection
```

Comment:
Czy tutaj nie musimy zupdate'ować `requirements.txt`?

Reply:
dokładnie, ale to tylko lokalnie, wiec ez, nie trzeba robic image

Reply:
done